src/compiler/crystal/command/spec.cr

```diff
@@ -8,14 +8,28 @@
 # directory, which usually is just `require "spec"` but could
 # be anything else (for example the `minitest` shard).
 
+# Gain access to OptionParser for spec runner to include it in the usage
+# instructions.
+require "spec/cli"
+
 class Crystal::Command
   private def spec
     compiler = new_compiler
     link_flags = [] of String
     OptionParser.parse(options) do |opts|
-      opts.banner = "Usage: crystal spec [options] [files]\n\nOptions:"
+      opts.banner = "Usage: crystal spec [options] [files] [-- runtime_options]\n\nOptions:"
```

Comment:
The runtime_options can be passed before the file, there is no need to use `--` to split them. Wouldn't that be the preferred way to advertise them?

In that line of thought I would call the "Spec runner additional options" or somehing like that. 
I find it confusing that they are presented as "Runtime options"

Reply:
Yes, the `--` can be removed. I'd still leave it as `[options] [files] [runtime_options]` even though they can be mixed. Don't think there's a better way to do this and this makes at least sense from a structural PoV.

Naming is particularly hard with this one. I'm not convinced `Spec runner additional options` improves it, though. I would have more questions about that than `Runtime options` which is really just a factual description. Those are the options evaluated at runtime by the spec runner, the others are compiler options.